aioesphomeapi/model.py

```diff
@@ -643,6 +652,7 @@ class ClimatePreset(APIIntEnum):
 
 @_frozen_dataclass_decorator
 class ClimateInfo(EntityInfo):
+    feature_flags: int = 0
```

Comment:
We can add a compat call like we have for other infos similar to supported_presets_compat and others

Reply:
https://github.com/kbx81/aioesphomeapi/blob/73d44ccdd9d110da56f8ef16faf149dbd433c376/aioesphomeapi/model.py#L234

Reply:
Oh! Great catch! Yes, I'll do that -- it'll simplify the HA side. 😄

Reply:
Ok, I think I got this in [`faec940` (#1376)](https://github.com/esphome/aioesphomeapi/pull/1376/commits/faec940a389091ccc76a29e3e86ad5a3b65c2a58)